### PR TITLE
Add blog post "Demystifying Gaia-X Credentials"

### PR DESCRIPTION
**Pull request by markus-hentsch**

### CODE REVIEW

**Comment by markus-hentsch** [comment]
@martinmo can you try the local preview instructions at https://github.com/SovereignCloudStack/website/issues/952#issuecomment-2149974489 and check if you can review the PR using this method?

**Comment by markus-hentsch** [comment]
Note to self: I learned some new things about VC subject IDs here: https://github.com/SovereignCloudStack/gx-credential-generator/issues/70#issuecomment-2188235936

I'll need to update the blog post draft:

- [ ] emphasize that VC IDs need to be unique as per https://docs.gaia-x.eu/technical-committee/identity-credential-access-management/22.10/credential_format/#identifiers
- [ ] emphasize that in the VP, the Participant VC's legalRegistrationNumber reference ID must match the subject ID of the LRN VC in the same document
- [ ] rephrase the part that states that the LRN VC ID must be the hosted URL to the VC JSON (it does not seem to be the case after all)

**Comment by markus-hentsch** [comment]
> Note to self: I learned some new things about VC subject IDs here: [SovereignCloudStack/gx-credential-generator#70 (comment)](https://github.com/SovereignCloudStack/gx-credential-generator/issues/70#issuecomment-2188235936)
> 
> I'll need to update the blog post draft:
> 
>     * [x]  emphasize that VC IDs need to be unique as per https://docs.gaia-x.eu/technical-committee/identity-credential-access-management/22.10/credential_format/#identifiers
> 
>     * [x]  emphasize that in the VP, the Participant VC's legalRegistrationNumber reference ID must match the subject ID of the LRN VC in the same document
> 
>     * [x]  rephrase the part that states that the LRN VC ID must be the hosted URL to the VC JSON (it does not seem to be the case after all)

After concluding my research on the identifier topic, I added two new figures and corresponding paragraphs to explain the identifier relations, restrictions and possible approaches.

**Comment by martinmo** [comment]
> @martinmo can you try the local preview instructions at [#952 (comment)](https://github.com/SovereignCloudStack/website/issues/952#issuecomment-2149974489) and check if you can review the PR using this method?

It doesn't work for me – when I run

    bundle exec jekyll serve config _config.yml,_config.dev.yml  --incremental -H 0.0.0.0

in the scs-website container, I get a `Liquid Exception`:

```
  Liquid Exception: wrong number of arguments (given 1, expected 0; required keywords: type, args) in /site/_layouts/post.html
                    ------------------------------------------------
      Jekyll 4.3.3   Please append `--trace` to the `serve` command 
                     for any additional information or backtrace. 
                    ------------------------------------------------
/site/_gems/jekyll-assets/lib/jekyll/assets/extensible.rb:107:in `for?': wrong number of arguments (given 1, expected 0; required keywords: type, args) (ArgumentError)
	from /site/_gems/jekyll-assets/lib/jekyll/assets/proxy.rb:55:in `block in proxies_for'
	from /site/_gems/jekyll-assets/lib/jekyll/assets/proxy.rb:54:in `select'
	from /site/_gems/jekyll-assets/lib/jekyll/assets/proxy.rb:54:in `proxies_for'
	from /site/_gems/jekyll-assets/lib/jekyll/assets/proxy.rb:29:in `proxy'
	from /site/_gems/jekyll-assets/lib/jekyll/assets/tag.rb:198:in `internal'
	from /site/_gems/jekyll-assets/lib/jekyll/assets/tag.rb:66:in `render_raw'
	from /site/_gems/jekyll-assets/lib/jekyll/assets/tag.rb:82:in `render'
	from /usr/local/bundle/gems/liquid-4.0.4/lib/liquid/block_body.rb:103:in `render_node_to_output'
	from /usr/local/bundle/gems/liquid-4.0.4/lib/liquid/block_body.rb:91:in `render'
        ...
```

But I was able to follow a tip by Matthias and deployed your blog post on the staging website: https://staging.scs.community/website/2024/06/05/demystifying-gaia-x-credentials/. Here's a screenshot of the steps which achieve this:

Screenshot from 2024-06-26 14-32-08

**Review by martinmo — APPROVED** [review]
This is very well written and makes a complex topic accessible. I especially like how the use case is exercised step by step. Together with the nice graphics I was able to follow and understand.

Content-wise I don't have any complaints at all and I couldn't find typos or grammar issues. I only have two very minor suggestions which you may consider but in any case I already approve.

PS. Before merging, don't forget to rename the file name (and refresh+rename symlink from the `_i18n/de` folder) to update the publish date, which is still at 2024-06-05.

File: _i18n/en/_posts/blog/2024-06-05-demystifying-gaia-x-credentials.md [hunk 1]
```diff
@@ -0,0 +1,593 @@
...
+}
+```
+
+Note: It is not mandatory to use the `/.well-known/` directory here.
+In contrast to the DID document, webserver, path and filename can be arbitrary here as long as we host the resulting Verifiable Credential at this address later on.
+This is true for all the Verifiable Credentials that we will create in the following sections.
+
+The Notarization API supports different types of Legal Registration Numbers, including VAT ID which our example uses.
+
+In the response to this request we receive our first Verifiable Credential from the Notarization API in JSON format.
```
[the remark this answers is not in the document]
Comment (markus-hentsch) [reply _i18n/en/_posts/blog/2024-06-05-demystifying-gaia-x-credentials.md:258]: Good point. I'll see if I can add a representative example.

File: _i18n/en/_posts/blog/2024-06-05-demystifying-gaia-x-credentials.md [hunk 2]
```diff
@@ -0,0 +1,593 @@
...
+layout: post
+title: "Demystifying Gaia-X Credentials"
+author:
+  - "Anja Strunk"
+  - "Markus Hentsch"
+avatar:
+  - "avatar-candh.jpg"
+---
+
+## Generating compliant Verifiable Credentials using the GXDCH
```
[the remark this answers is not in the document]
Comment (markus-hentsch) [reply _i18n/en/_posts/blog/2024-06-05-demystifying-gaia-x-credentials.md:11]: Fair point. I'll have a try and see how it looks when removing this and moving all other sections up by one level.

**Comment by markus-hentsch** [comment]
@martinmo thanks for your review! Based on your feedback I did some adjustments: https://github.com/SovereignCloudStack/website/compare/d974bb1bcf49a225fe951a0c65cdae8df688ebef...6b4f0b06198c7b18ab563f3ce801f36ade8dd993

I redeployed at: https://staging.scs.community/website/2024/06/05/demystifying-gaia-x-credentials/

I'll update the date once we make the final publishing decision (especially regarding the ongoing work on the scs-did-creator and gx-credential-generator tools).

File: _i18n/en/_posts/blog/2024-06-05-demystifying-gaia-x-credentials.md [hunk 3]
```diff
@@ -0,0 +1,646 @@
...
+        "created": "2024-06-25T14:38:38.916Z",
+        "proofPurpose": "assertionMethod",
+        "verificationMethod": "did:web:registration.lab.gaia-x.eu:v1#X509-JWK2020",
+        "jws": "<signature>"
+    }
+}
+```
+(the JWS signature field has been replaced by the placeholder `<signature>` for readability)
+
+We will put this JSON file on our web server at the path we specified during the request, i.e. `mydomain.com/.well-known/lrn.json`.
```
Comment (martinmo) [inline _i18n/en/_posts/blog/2024-06-05-demystifying-gaia-x-credentials.md:312]:
```suggestion
We will put this JSON file on our web server at the path we specified during the request, i.e., `mydomain.com/.well-known/lrn.json`.
```

**Review by martinmo — APPROVED** [review]

File: _i18n/en/_posts/blog/2024-06-05-demystifying-gaia-x-credentials.md (hunk 3 above)
Comment (markus-hentsch) [reply _i18n/en/_posts/blog/2024-06-05-demystifying-gaia-x-credentials.md:312]: TIL: https://english.stackexchange.com/a/6896

**Comment by anjastrunk** [comment]
Thank's @markus-hentsch for finalizing this blog post. It reached a very good level and IMO will be a value-added to all Gaia-X users!

**Review by fkr — APPROVED** [review]
Awesome post.
Even though I did not fully verified all technical details, I'll approve it, since @markus-hentsch and @anjastrunk are the experts here.

**Pull request merged**